cypress/integration/RadioGroup.spec.ts

```diff
@@ -0,0 +1,29 @@
+import * as h from '../helpers';
+
+describe('Radio Group', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Default', 'Alert', 'Error with Grow'].forEach(story => {
+    context(`given the '${story}' story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Components|Inputs/Radio/React/Top Label/Radio Group', story);
+      });
+
+      it('should pass accessibility checks', () => {
+        cy.checkA11y();
```

Comment:
I asked Accessibility this question a few months ago. They estimate `axe-core` catches around 13% of accessibility issues. They think we should run it because it is so cheap to do so, but not treat it as a replacement for accessibility specifications.

What you describe should be specifications in our Cypress files.

```ts
describe('given the Error story is rendered', () => {
  it('should have a legend element', () => {
    // get radio input
    // .parents('legend').should('exist')
  })

  it('should have an aria-describedby linking to the input element', () => {
    // find radio input and get id to check against the value of the `aria-describedby`
  })
})
```

I think the test block description is misleading. It should say something like:

```ts
it('should pass axe-core checks', () => {
```

To clarify:
`cy.checkA11y()` is _not_ a replacement for accessibility specifications. We simply treat it as a black box and not care if we duplicate accessibility checks.